spotify/models/user.py

```diff
@@ -122,23 +203,27 @@ def _raise(*args, **kwargs):
 
         return [klass(self.__client, item) for item in resp["items"]]
 
-    async def _refreshing_token(self, expires: int, token: str):
+    async def _refreshing_token(self, expires: int, token: str) -> None:
         while True:
             await asyncio.sleep(expires - 1)
 
-            client_id = self.client.http.client_id
-            client_secret = self.client.http.client_secret
+            self.token_info = await _refresh_token(self.client, token)
```

Comment:
Why shouldn't `_refresh_token` be dissolved into this method body?

Reply:
The same logic is needed when initializing the User object solely from a refresh token, e.g. from cache when the token has expired or directly by providing it (see `from_token_info`.

I moved the method to avoid duplicate code.